CPX_Simon_Game/code.py

```diff
@@ -0,0 +1,106 @@
+import time
+import random
+from adafruit_circuitplayground.express import cpx
+
+cpx.pixels.brightness = 0.1  # adjust NeoPixel brightness to your liking
+
+REGION_LEDS = [
+    [5, 6, 7],  # yellow region
+    [2, 3, 4],  # blue region
+    [7, 8, 9],  # red region
+    [0, 1, 2],  # green region
+]
+
+REGION_COLOR = [
```

Comment:
ditto

Reply:
Limor, the code works and the guide is ready. Would you like me to have the author make the tuple changes? He'll probably have to do it after work and then test it.

Reply:
@ladyada - code now lints, shall I ask the author to make the tuple changes?

Reply:
please do it for them and let them know you did it

Reply:
Right-o